test/container_test.py

```diff
@@ -111,7 +217,7 @@ def test_set_volume_unit_conv(self):
         c.well(1).set_volume(".1:milliliter")
         self.assertTrue(c.well(1).volume == Unit(100, "microliter"))
         with self.assertRaises(ValueError):
-            c.well(2).set_volume("1:liter")
+            c.well(2).set_volume("1:milliliter")
```

Comment:
nope, either way is fine

I see what you mean, we should probably add tests for utils.py